src/lib.rs

```diff
@@ -1,6 +1,12 @@
 #![no_std]
 
+#[allow(unused_imports)]
```

Comment:
```suggestion
```

Reply:
this is fix for a failed CI (it fails the `cargo clippy` check)